alpha/alpha-server/src/main/java/io/servicecomb/saga/alpha/server/AlphaConfig.java

```diff
@@ -44,15 +44,27 @@ TxEventRepository springTxEventRepository(@Value("${alpha.server.port:8080}") in
 
     TxEventRepository eventRepository = new SpringTxEventRepository(eventRepo);
 
-    ThriftStartable startable = new ThriftStartable(
+    ServerStartable startable = buildGrpc(port, omegaCallback, eventRepository);
+    CompletableFuture.runAsync(startable::start);
```

Comment:
if grpc service starts without blocking, we don't need to run it in another thread